src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs

```diff
@@ -350,7 +431,7 @@ public bool IsSpecificFramework
         /// <summary>
         /// True if this framework is Net5 or later, until we invent something new.
         /// </summary>
-        internal bool IsNet5Era { get; set; }
+        public bool IsNet5Era { get; private set; }
```

Comment:
Can we please keep this internal? 

This is a very low level API and I'd like us to keep it as clean as possible. 
Personally I'd rather have code duplication than expose a concept that's really a consequence of a business decision. 

@zivkan thought?

Reply:
I missed that this changed. I agree, we should keep it internal. If we get feedback from the SDK team or others that they hate having to do the TFI & TFV comparisons themselves and duplicating this check, we can add it without it being a breaking change (and also try to come up with a better name)

Reply:
Fixed :)